dask_cuda/utils.py

```diff
@@ -649,3 +654,144 @@ def get_gpu_uuid_from_index(device_index=0):
     pynvml.nvmlInit()
     handle = pynvml.nvmlDeviceGetHandleByIndex(device_index)
     return pynvml.nvmlDeviceGetUUID(handle).decode("utf-8")
+
+
+def get_worker_config(dask_worker):
+    # assume homogenous cluster
+    plugin_vals = dask_worker.plugins.values()
+    ret = {}
+
+    # device and host memory configuration
+    for p in plugin_vals:
+        ret[type(p).__name__] = {
```

Comment:
Looking again (sorry!), shall we indicate that these are plugins? `ret[f"[plugin] {type(p).__name__}"] = ...` ?

Reply:
I'm happy to include it and we can continue iterating.  I doubt most users deeply care about what is label a plugin but we (developers/anyone helping to debug) may find it useful.